tests/unit/Configuration.unitTest.ts

```diff
@@ -23,39 +22,39 @@ describe("ConfigurationUtil", () => {
                 const emptyConfig = new Configuration("../../tests/resources/EmptyConfig.yml");
                 try {
                     emptyConfig.getInvokeConfig();
-                    expect.fail();
+                    expect.assertions(1); // should have thrown an error, test failed
```

Comment:
This needs to be outside the try/catch, as it will never be reached if the tests proceeds as expected.

Reply:
Preferably before

Reply:
That's my change. Isn't it it supposed to make the test break if the previous call completes successfully?